README.md

```diff
@@ -29,6 +29,13 @@ PhysicsTutorials.open_notebooks()
 Supported source files for tutorials are Jupyter notebooks, Weave.jl files, or Literate.jl files.
 To contribute a tutorial, clone the repository and put the source file into `tutorials/<category>/<tutorial_name>/` and name it `<tutorial_name>.ipynb` (extension `.jmd`/`.jl` for Weave/Literate sources). To trigger the generation process of all output formats, run the following code from within the repository root folder:
 
+0. Make sure you have `Fire` installed, if not, install it with `julia>]add Fire`,
```

Comment:
Perhaps simply `] add Fire` to allow for copy and paste?

Reply:
Here, a space is missing, `julia> ] add Fire` behaves the same as `] add Fire` when one copys it to REPL.
Here, the problem is, both of them does not allow copy and paste?

Reply:
> Here, the problem is, both of them do not allow copy and paste?

Why? I don't see the problem.

Reply:
On my laptop, it does not enter pkg mode, instead, it prints a `]`. My system is Ubuntu 18.04.

Reply:
For me (Windows 10), pasting `] add Fire` switches to pkg mode (desired behavior) while pasting `julia> ] add Fire` does not work:

```julia
julia> julia> ] add Fire
ERROR: syntax: unexpected "]"
```

Anyways, I feel `] add Fire` is our best bet here. I'll merge and change this myself afterwards.